config/routes.rb

```diff
@@ -8,7 +8,10 @@
     post :global_validity_check, on: :member
   end
 
-  resources :system_messages, only: %i[new create show]
+  resources :system_messages, only: %i[new create show] do
+    post 'twilio', format: 'xml', on: :member
```

Comment:
I think this is just used for robo calls, please check

Reply:
removed